bin/utils/print-event-command-utils.js

```diff
@@ -49,9 +49,7 @@ function printLocalCommandPush(
       pkg.bin
     )} -df '${getDefinitionFile()}' build ${eventFlowTypeToCliFlowType(
       getFlowType()
-    )} -p ${eventData.repository.full_name} -b ${eventData.ref
```

Comment:
this is nothing a missing change from previous PR, `-p` argument was removed at some point but log still printing it.